src/components/Header/Header.tsx

```diff
@@ -1,9 +1,73 @@
-import React, {FC, JSX} from "react";
+import React, {FC, JSX, useEffect, useState} from "react";
+import SvgSprite from "../reusable/svgSprite/SvgSprite";
+import './header.scss';
 
+type TNav = {
+    title: string,
+    id: string
+}
 const Header: FC = (): JSX.Element => {
+    const [dataLinks, setDataLinks] = useState<TNav[]>();
+
+    const getNavigationLinks = () => {
+        fetch(`${process.env.REACT_APP_API_URL}/navigationInfo`)
+            .then(response => response.json())
+            .then(json => setDataLinks(json))
+            .catch(error => console.log(error))
+    }
+
+    const scroll = (offset: number) => {
+        window.scrollTo({
+            top: offset,
+            behavior: "smooth",
+        });
+    }
+
+    const smoothScroll = (targetId: string) => {
+        if (targetId) {
+            if(targetId === '#') {
```

Comment:
in this case I would better add an anchor to top section and navigate to it instead of having special logic with offset 0